src/systems/systems/src/air_conditioning/mod.rs

```diff
@@ -0,0 +1,387 @@
+        }
+        Self {
+            pack_1_pb: OnOffFaultPushButton::new_on(context, "COND_PACK_1"),
+            pack_2_pb: OnOffFaultPushButton::new_on(context, "COND_PACK_2"),
+            temperature_selectors,
+        }
+    }
+
+    fn selected_cabin_temperatures(&self) -> HashMap<&'static str, ThermodynamicTemperature> {
+        let mut temperature_selectors_values: HashMap<&'static str, ThermodynamicTemperature> =
+            HashMap::new();
+        for (id, knob) in &self.temperature_selectors {
+            temperature_selectors_values.insert(
+                id,
+                // Map from knob range 0-100 to 18-30 degrees C
+                ThermodynamicTemperature::new::<degree_celsius>(knob.value() * 0.12 + 18.),
+            );
+        }
+        temperature_selectors_values
+    }
+
+    fn pack_1_pb_is_on(&self) -> bool {
+        self.pack_1_pb.is_on()
+    }
+
+    fn pack_2_pb_is_on(&self) -> bool {
+        self.pack_2_pb.is_on()
+    }
+}
+
+impl SimulationElement for AirConditioningSystemOverhead {
+    fn accept<T: SimulationElementVisitor>(&mut self, visitor: &mut T) {
+        for selector in self.temperature_selectors.values_mut() {
+            selector.accept(visitor);
+        }
+        self.pack_1_pb.accept(visitor);
+        self.pack_2_pb.accept(visitor);
+
+        visitor.visit(self);
+    }
+}
+
+impl DuctTemperature for AirConditioningSystem {
+    fn duct_demand_temperature(&self) -> HashMap<&'static str, ThermodynamicTemperature> {
+        self.acsc.duct_demand_temperature()
+    }
+}
+
+impl PackFlow for AirConditioningSystem {
+    fn pack_flow(&self) -> MassRate {
+        self.acsc.pack_flow()
+    }
+}
+
+struct PackFlowValve {
+    pack_flow_valve_id: VariableIdentifier,
+
+    number: usize,
+    is_open: bool,
+    timer_open: Duration,
+}
+
+impl PackFlowValve {
+    fn new(context: &mut InitContext, number: usize) -> Self {
+        Self {
+            pack_flow_valve_id: context.get_identifier(Self::pack_flow_valve_id(&number)),
+            number,
+            is_open: false,
+            timer_open: Duration::from_secs(0),
+        }
+    }
+
+    fn pack_flow_valve_id(number: &usize) -> String {
+        format!("COND_PACK_FLOW_VALVE_{}_IS_OPEN", number)
+    }
+
+    fn update(&mut self, context: &UpdateContext, open_fcv: &impl FlowControlValveSignal) {
+        self.is_open = open_fcv.should_open_fcv()[self.number - 1];
+        if self.is_open {
+            self.timer_open += context.delta();
+        } else {
+            self.timer_open = Duration::from_secs(0);
+        }
+    }
+
+    fn fcv_timer(&self) -> Duration {
+        self.timer_open
+    }
+
+    fn fcv_is_open(&self) -> bool {
+        self.is_open
+    }
+}
+
+impl SimulationElement for PackFlowValve {
+    fn write(&self, writer: &mut SimulatorWriter) {
+        writer.write(&self.pack_flow_valve_id, self.is_open);
+    }
+}
+
+pub struct Air {
```

Comment:
Yes, the idea is that this will be the elemental type that will be used to simulate the air moving through the packs and mixing unit, so more elements of the Air Conditioning system will use it in the future.